### DISCUSSION TITLE
Download dataset, example run, ..., etc.

### DISCUSSION THREAD

**Discussion by hungyiwu** [Borda/BIRL#52] [Q&A] [answered]
#### What is your question?
I tried to run the benchmarks but got some issues:
1. Submitted dataset request at http://cmp.felk.cvut.cz/~borovji3/?page=dataset but no reply
2. Followed the main `README.md` of `master` branch but failed due to some parameter name change, for example `-nb` does not work anymore, likely changing to `-n`
3. Out of curiosity, ran `pytest -v` but it failed (stuck at some multi-thread call forever)
4. Found this [post](https://forum.image.sc/t/automatic-non-rigid-histological-image-registration-anhir-challenge/21492/2) on `image.sc` forum but the [download link](https://anhir.grand-challenge.org/Download/) said `Forbidden`

Is this repo still alive and is it possible to run the benchmarks now, given that the challenge is likely ended?

**Comment by Borda** [comment, marked as answer]
Thank you for your interest

> 1. Submitted dataset request at http://cmp.felk.cvut.cz/~borovji3/?page=dataset but no reply

The email sending is automatical and instant, is possible that it fall in spam or you made a typo? 

> 2. Followed the main `README.md` of `master` branch but failed due to some parameter name change, for example `-nb` does not work anymore, likely changing to `-n`

It is possible that Readme may be outdated, mind send a PR with corrections? 

> 3. Out of curiosity, ran `pytest -v` but it failed (stuck at some multi-thread call forever)

I am jsut running some test for #51 so I will check it 

> 4. Found this [post](https://forum.image.sc/t/automatic-non-rigid-histological-image-registration-anhir-challenge/21492/2) on `image.sc` forum but the [download link](https://anhir.grand-challenge.org/Download/) said `Forbidden`

You need following instructions on that site, so you have to register and join the challenge to see full contebt

> Is this repo still alive and is it possible to run the benchmarks now, given that the challenge is likely ended?

Yes, I have updated the evaluation image a few weeks ago so this is fine...

**Comment by hungyiwu** [comment]
Thank you for your fast reply!
1. I just sent the data request again and double-checked any typo but still no email; confirmed not in spam
2. Will make PR once successfully ran the command
3. Thank you for checking!
4. Confirmed seeing the download links after registering with Grand Challenges, thank you!
5. I'm so glad to know this is alive, thank you for your effort and I'll see what I can help here!

**Comment by Borda** [comment]
@hungyiwu just pushed docker image with installed all registration methods, check out https://hub.docker.com/r/borda/birl

**Comment by Borda** [comment]
ok, just tried and confirm that the automatic mail sending stopped working, I'll try to fix it... :]
http://cmp.felk.cvut.cz/~borovji3/index.php?page=dataset

**Comment by Borda** [comment]
the mail sending is fixed now, still, PR is welcome... :]

**Comment by hungyiwu** [comment]
Thank you!
